### DISCUSSION TITLE
Difference between organization domain and mailserver domain

### DISCUSSION THREAD

**Discussion by wh4u** [postalserver/postal#1454] [Help with using Postal]
At my postal setup i have multiple organizations and domains.
All domains have been inserted postal login -> select organization -> mailserver -> domains.

What is the difference on organization level domains and mailserver domains?

**Comment by rccess** [comment]
organization level domains can be used any mailserver deployed under the same organization
and mailserver level domain can be used by only the that specific mailserver only
